internal/sops/awskms/keysource.go

```diff
@@ -224,15 +227,25 @@ func NewMasterKeyFromArn(arn string, context map[string]string, awsProfile strin
 
 // createKMSConfig returns a Config configured with the appropriate credentials.
 func (key MasterKey) createKMSConfig() (*aws.Config, error) {
-	// Use the credentialsProvider if present, otherwise default to reading credentials
-	// from the environment.
+	re, err := regexp.Compile(arnRegex)
```

Comment:
`MustCompile` is probably a better option.

Reply:
IMO we shouldn't call a function that can panic, even if the argument is a constant.

Reply:
The constant is expected to be valid, panic'ing here is thus absolutely justified?

Reply:
Yes fair point, I try to avoid calling functions that can panic, when they have a non-panic alternative, but I guess it's okay to use it here.